prow/config/plugins.yaml

```diff
@@ -102,6 +102,11 @@ repo_milestone:
 milestone_applier:
   ti-community-infra/tichi:
     master: v1.4
+  pingcap/tics:
+    release-4.0: v1.4
+    release-5.0: v1.4
+    release-5.1: v1.4
+    release-5.2: v1.4
```

Comment:
I'm not sure I understand it correctly. PTAL @Mini256

Reply:
> The milestoneapplier plugin automatically applies the configured milestone for the base branch after a PR is merged. If a PR targets a non-default branch, it also adds the milestone when the PR is opened.

You should add the corresponding milestone name, refers to [milestones](https://github.com/pingcap/tics/milestones?state=open) page:

```yaml
pingcap/tics:
  release-4.0: v4.0.15
  release-5.0: v5.0.4
  release-5.1: v5.1.2
  release-5.2: v5.2.0
```

But I am not sure whether the milestone on the current release-5.0 branch is v5.0.4 or v5.0.5.

Reply:
The robot will automatically add a milestone according to the configuration:

```yaml
milestone_applier:
  ti-community-infra/tichi:
    master: v1.4
```

For example on the master branch: https://github.com/ti-community-infra/tichi/pull/455

![image](https://user-images.githubusercontent.com/5086433/129507530-e1d8a03b-87db-4ae3-903d-fd4312ecc5f1.png)

Reply:
So every time we need to apply a new milestone, we need to file a PR to update the plugins.yaml in this repo? Then the milestone_applier seems useless to me...

Reply:
What is the effect you want to achieve?